swagger.yaml

```diff
@@ -4899,6 +4899,11 @@ definitions:
         x-nullable: true
         type: string
         description: A string which will be used as Authorization Bearer token to fetch the ignition from ignition_endpoint_url.
+      ignition_endpoint_http_headers:
+        x-go-custom-tag: gorm:"type:text"
+        type: string
```

Comment:
I suggest modeling this the same as the node_label property right below this one

Reply:
Thank you! I've updated it to resemble node_label, how does it look?